wiki/Manual.md

```diff
@@ -3111,7 +3111,7 @@ $exchange = new $exchange_class (array (
 Note that your private requests will fail with an exception or error if you don't set up your API credentials before you start trading. To avoid character escaping **always write your credentials in single quotes**, not double quotes (`'VERY_GOOD'`, `"VERY_BAD"`).
 
 #### API Key Permissions
-When you get errors like `"Invalid API-key, IP, or permissions for action."`, then, most likely, the problem is not within ccxt, please avoid opening issues unless you ensure that:
+When you get errors like `"Invalid API-key, IP, or permissions for action."` or `"API-key format invalid"`, then, most likely, the problem is not within ccxt, please avoid submitting a new support ticket unless you ensure that:
```

Comment:
"support ticket" is more generic and suitable imo (i.e. github, discord, telegram, etc..)

Reply:
@ttodua actually I think that "support ticket" is a bit misleading, because it seems the users have to open a support ticket using helpdesk or some other platform like that

Reply:
ok, replaced by `support request`, is that ok? or however you think, plz feel free (as always) to edit-in per your thought

Reply:
maybe now it reads better.

Reply:
@carlosmiei I have adjusted, lmk